index.html

```diff
@@ -11,18 +11,184 @@
+              <label class="input-label" for="website">Website address</label>
+              <div class="relative-el">
+                <input type="text" class="fullwidth-input" name="website"
+                  id="website" placeholder="Website url"
+                  aria-labelledby="website" aria-required="true"
+                  tabindex="0"
+                  required="required" />
+                <label for="website">
+                  <span id="website-input-suffix">.free-websites.com</span>
+                </label>
+              </div>
+              <!-- error handling -->
+            </div>
+          </li>
 
+          <li>
+            <h2>Create an account.</h2>
+            <div>
+              <div id="fullname-holder">
+                <label class="input-label" for="fullname">Full name</label>
+                <input type="text" class="fullwidth-input" name="fullname"
+                  id="fullname" placeholder="Full name"
+                  tabindex="0" required="required"
+                  aria-labelledby="fullname" aria-required="true" />
+                <!-- error handling -->
+              </div>
+              <div id="email-holder">
+                <label class="input-label" for="emailaddress">
+                  Email address</label>
+                <input type="email" id="emailaddress" class="fullwidth-input"
+                  name="email" placeholder="Email"
+                  aria-labelledby="emailaddress" aria-required="true"
+                  required="required" tabindex="0" />
+                <!-- error handling -->
+              </div>
+              <div id="password-holder">
+                <label class="input-label" for="password">Password</label>
+                <input type="password" id="password" class="fullwidth-input"
+                  name="password" placeholder="Password"
+                  aria-labelledby="password" aria-required="true"
+                  required="required" tabindex="0" />
+                <label id="showpassword-label">
+                  <input type="checkbox" id="showpassword" aria-checked="false"
+                    name="showpassword" aria-hidden="true"
+                    aria-labelledby="reveal-description" />
+                  <span id="reveal-description">Reveal password</span>
+                </label>
+                <!-- error handling -->
+              </div>
+            </div>
+          </li>
 
+          <li>
+            <h2>Enter your payment info.</h2>
+            <div>
+              <div id="creditcardnumber-holder">
+                <label class="input-label" for="creditcardnumber">
+                  Credit card number</label>
+                <input type="text" name="creditcardnumber"
+                  class="fullwidth-input"
+                  id="creditcardnumber" placeholder="Credit card number"
+                  aria-labelledby="creditcardnumber" aria-required="true"
+                  required="required" tabindex="0" /><!--
```

Comment:
I've added this so the input and div don't have a space between them. If you remove this comment a space is added and it isn't responsive.